docs/RunningFirstBenchmark.md

```diff
@@ -0,0 +1,114 @@
+# Running Your First Benchmark
+
+This tutorial is aimed at people new to the library and shows you how to run benchmarks with the Google Benchmark library.
+
+# Install the prerequisites
+
+We need the following programs:
+
+* git : to clone google benchmark and gtest libraries
+* cmake : to build google benchmark
+* make : to build google benchmark. Advanced users can use an alternative cmake backend like ninja.
+* g++ : to compile your c++ code. Advanced users can use other compilers like clang instead.
+
+On Fedora 28, the following command installs the requirements:
+
+```
+sudo dnf install --assumeyes wget git-core cmake make gcc-c++
+```
+
+At the time of writing, the versions were as follows:
+* g++ : 8.1.1 20180502
+* cmake : 3.11.2
+* gnu make: 4.2.1
+* git: 2.17.1
```

Comment:
I think it's better to avoid this specificity and just have the users install the packages however they know how. there are too many platforms and variants to go into detail.